src/components/search/utils.ts

```diff
@@ -0,0 +1,26 @@
+import { Fetcher } from 'swr';
```

Comment:
Can we move this file to requests ?

Reply:
So I think the actual fetch logic we will move to `requests` when we implement it, so like on line 19 we would just call a function imported from `requests`. But since this `Fetcher` type is specific to the `swr` hook, and since we're only using that in the search component, I think I'd still want the definition of `fetcher` to remain here.

Reply:
okay, sure.